src/phpDocumentor/Reflection/Php/NodesFactory.php

```diff
@@ -59,9 +59,9 @@ public function __construct(Parser $parser, NodeTraverser $traverser)
      *  ParserFactory::PREFER_PHP5, ParserFactory::ONLY_PHP7 or ParserFactory::ONLY_PHP5
      * @return static
      */
-    public static function createInstance($kind = ParserFactory::PREFER_PHP7)
+    public static function createInstance($kind = ParserFactory::ONLY_PHP7): self
```

Comment:
not sure if this is the right way, this means that we are not able to parse php5 code from now.